sections/browsers.include

```diff
@@ -282,7 +282,7 @@
 
 <h4 id="browsing-context-security">Security</h4>
 
-  A <a>browsing context</a> <var>A</var> is <dfn>familiar
+  A <a>browsing context</a> <var>A</var> is <dfn lt="familiar with|familiar">familiar
```

Comment:
If you make sure the whole `dfn` element is on one line you should not need to specify "familiar with" in the lt attribute, no?

Reply:
the term is also used as "familiar" in some paragraphs, so I use  "lt='...'" for the text needs to be in a conjugated form... any suggestion?

Reply:
@chaals The links of "familiar with" wont work once we remove "familiar with" from "lt", so I keep that and remove the line break in the latest commit.